app/components/statistic-list/bar/component.js

```diff
@@ -0,0 +1,17 @@
+import Component from "@glimmer/component";
+
+export default class StatisticListBar extends Component {
+  get didFinishEffortsInBudget() {
+    if (this.args.remaining === 0 && this.args.value < this.args.goal) {
+      return true;
+    }
+    return false;
+  }
```

Comment:
We need a special flag `archived` on the task / project to realize this feature. This has to be done in the backend. Otherwise all running tasks, which don't have remaining effort defined are colored :green_circle:, which is not intended.

Let's skip the :green_circle: state for now. We have to discuss the efforts for this first.